bitnami/wordpress/6/debian-11/rootfs/opt/bitnami/scripts/libwordpress.sh

```diff
@@ -583,6 +591,29 @@ if ( !defined( 'WP_CLI' ) ) {
 EOF
 }
 
+########################
+# Disable access to the WordPress XML-RPC endpoint
+# Globals:
+#   *
+# Arguments:
+#   $1 - path to .htaccess file
+# Returns:
+#   None
+#########################
+wordpress_disable_xmlrpc_endpoint() {
```

Comment:
Nice! But you have to bear in mind that we also offer the option to use wordpress with nginx (see [bitnami/wordpress-nginx](https://github.com/bitnami/containers/tree/main/bitnami/wordpress-nginx)). The `libwordpress.sh` should have logic that can be used in both implementations, so you should also take that in mind.

Reply:
Well the issue with this is, that nginx doesn't support .htaccess files so the alternative would be to add a nginx config:
```nginx
location = /xmlrpc.php {
    deny all;
}
```

Would this be sufficient?

Reply:
Yes, I think it would be enough 😁

Reply:
Ok, wordpress-nginx now also has support for this optiom. 
I also moved the the code from .htaccess to the apache server config so that the code for the nginx and apache images have a similiar structure.